src/api/app/views/webui2/shared/_repositories.html.haml

```diff
@@ -31,7 +31,6 @@
       locals: { flags: flags[:useforbuild], project: project, package: package,
                 architectures: architectures, table_id: 'flag_table_useforbuild' }
 
-Hallo #{user_can_modify}
```

Comment:
:rofl:

Reply:
Let's put this in the funny fail of the year 2019.